sdk/cosmosdb/cosmos/src/client/ChangeFeed/ChangeFeedForEpkRange.ts

```diff
@@ -378,7 +382,11 @@ export class ChangeFeedForEpkRange<T> implements ChangeFeedPullModelIterator<T>
     feedRange: ChangeFeedRange,
     diagnosticNode: DiagnosticNodeInternal,
   ): Promise<ChangeFeedIteratorResponse<Array<T & Resource>>> {
-    const feedOptions: FeedOptions = { initialHeaders: {}, useIncrementalFeed: true };
+    const feedOptions: FeedOptions = {
+      initialHeaders: {},
+      useIncrementalFeed: true,
```

Comment:
Can we name this **useLatestVersions**? We are moving away from _Incremental_ and toward _LatestVersions_. Just do this everywhere so I do not have to write a review for every instance.

Reply:
updated